<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SaveFile</class>
 <widget class="QWidget" name="SaveFile">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>284</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>284</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>640</width>
    <height>284</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Сохранение результата</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/img/logo.png</normaloff>:/img/logo.png</iconset>
  </property>
  <widget class="QPushButton" name="editFolder">
   <property name="geometry">
    <rect>
     <x>547</x>
     <y>44</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Изменить</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>142</y>
     <width>531</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>Поляризация:</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>183</y>
     <width>531</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Горизонтальная</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>237</y>
     <width>531</width>
     <height>28</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>28</height>
    </size>
   </property>
   <property name="text">
    <string>Сохранить</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="experimentName">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>113</y>
     <width>531</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="folderPath">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>44</y>
     <width>531</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string>C:\</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>3</y>
     <width>531</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>Папка назначения:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>73</y>
     <width>531</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Название эксперимента:</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>531</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Вертикальная</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
